backends/nxp/backend/neutron_operator_support.py

```diff
@@ -0,0 +1,77 @@
+# Copyright 2025 NXP
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+from executorch.backends.nxp.backend.neutron_target_spec import NeutronTargetSpec
+
+
+def is_tensor_invariant_permutation(
+    input_shape: list[int], permutation: list[int]
+) -> bool:
+    new_permutation = [
```

Comment:
NIT: Slightly longer, but perhaps better readable:
```suggestion
    def input_shape_is_not_one(perm_index):
        return input_shape[perm_index] > 1
    new_permutation = filter(input_shape_is_not_one, permutation)
```

Reply:
I like your change. Implemented